src/Dapr.Actors/Runtime/IActorReminder.cs

```diff
@@ -41,5 +41,11 @@ public interface IActorReminder
         /// </summary>
         /// <value>The user state passed to the reminder invocation.</value>
         byte[] State { get; }
+
+        /// <summary>
+        /// Gets the number of repetitions of the reminder left to be invoked.
+        /// </summary>
+        /// <value>The number of repetitions of the reminder left to be invoked.</value>
+        int RepetitionsLeft { get; }
```

Comment:
This is a binary breaking change, we cannot accept it, so we need to find another solution. Is there anywhere that this property is read?

Reply:
Its only read internally,